lib/components/Checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,268 @@
+                        theme.atoms.borderRadius.standard,
+                        theme.atoms.transition.fast
+                      )}
+                    />
+                    <Box
+                      backgroundColor="input"
+                      borderColor="formAccent"
+                      borderWidth="standard"
+                      className={classnames(
+                        styles.checkbox,
+                        styles.checkboxChecked,
+                        theme.atoms.backgroundColor.formAccent,
+                        theme.atoms.borderRadius.standard,
+                        theme.atoms.transition.fast
+                      )}
+                    />
+                    <Box
+                      backgroundColor="inputDisabled"
+                      borderWidth="standard"
+                      className={classnames(
+                        styles.checkbox,
+                        styles.checkboxDisabled,
+                        theme.atoms.borderRadius.standard,
+                        theme.atoms.transition.fast
+                      )}
+                    />
+                    <Box
+                      borderColor="critical"
+                      borderWidth="standard"
+                      style={{
+                        opacity: tone === 'critical' ? 1 : 0
+                      }}
+                      className={classnames(
+                        styles.checkbox,
+                        styles.checkboxCritical,
+                        theme.atoms.borderRadius.standard,
+                        theme.atoms.transition.fast
+                      )}
+                    />
+                    <TickIcon
+                      size="fill"
+                      className={classnames(
+                        styles.checkboxIcon,
+                        theme.atoms.fill.white,
+                        theme.atoms.transition.fast
+                      )}
+                    />
+                  </Box>
+                  <Text
+                    size="interaction"
+                    color={textColorForState({ disabled, hovered })}
+                    {...(checked && children ? { weight: 'strong' } : {})}
+                  >
+                    {label}
+                  </Text>
+                </Box>
+                {children ? (
+                  <Box
+                    paddingLeft={inChecklistCard ? 'gutter' : 'none'}
+                    paddingRight={inChecklistCard ? 'gutter' : 'none'}
+                  >
+                    <Box
+                      paddingLeft={inChecklistCard ? 'medium' : 'small'}
+                      paddingBottom="small"
+                      className={styles.children}
+                      style={{ marginLeft: px(checkboxSize) }}
+                    >
+                      {children}
+                    </Box>
+                  </Box>
+                ) : null}
+              </div>
+              {message !== false ? (
```

Comment:
Is this a tri-state? e.g. `true`, `false`, `undefined`?
If so can we represent that as an enum instead?

Reply:
Nope, it's either a string, or `false` to disable the message entirely.